testsuite/features/support/commonlib.rb

```diff
@@ -357,6 +357,9 @@ def extract_logs_from_node(node, host)
     elsif transactional_system?(host)
       node.run('transactional-update --continue -n pkg install tar')
     end
+    if deb_host?(node)
+      node.run('apt -y install xz')
+    end
```

Comment:
```suggestion
node.run('apt -y install xz') if deb_host?(node)
```
Could fix the RuboCop issue. See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/IfUnlessModifier. But my indentation is not correct.